geniusyield-market-maker/geniusyield-market-maker.cabal

```diff
@@ -149,9 +154,66 @@ executable geniusyield-market-maker-exe
     , random
     , random-fu
     , vector
+  ghc-options:      -Wall
+  default-language: Haskell2010
+  mixins:
+    geniusyield-market-maker:orderbook-annset requires (GeniusYield.OrderBot.DataSource as GeniusYield.OrderBot.DataSource.Providers)
 
-  hs-source-dirs:   . src
+-- Build targets
+
+executable geniusyield-market-maker-exe
+  import:           common-lang
+  import:           common-deps
+  import:           common-ghc-opts
+  main-is:          Main.hs
+  build-depends:
+      cardano-api
+    , deriving-aeson
+    , envy
+    , geniusyield-annset
+    , geniusyield-dex-api
+    , geniusyield-market-maker:{datasource-providers, orderbook-annset, geniusyield-market-maker-lib}
+    , geniusyield-orderbot-framework:common
+    , ghc-typelits-knownnat
+    , maestro-sdk
+    , MonadRandom
+    , plutus-core
+    , plutus-ledger-api
+    , ply-core
+    , random
+    , random-fu
+    , vector
+  hs-source-dirs:   .
   ghc-options:      -Wall
   default-language: Haskell2010
   mixins:
     geniusyield-market-maker:orderbook-annset requires (GeniusYield.OrderBot.DataSource as GeniusYield.OrderBot.DataSource.Providers)
+
+test-suite pproviders-status-sequence
+  import:           common-lang
+  import:           common-deps
+  import:           common-ghc-opts
+  type:             exitcode-stdio-1.0
+  hs-source-dirs:   test
+  main-is:          Main.hs
+  other-modules:
+    GeniusYield.Test.MarketMaker.MakerBot
+    GeniusYield.Test.MarketMaker.Utils
+  build-depends:    base
+                  , cardano-api
+                  , deriving-aeson
+                  , envy
+                  , geniusyield-annset
+                  , geniusyield-dex-api
+                  , geniusyield-market-maker:{datasource-providers, orderbook-annset, geniusyield-market-maker-lib}
+                  , geniusyield-orderbot-framework:common
+                  , ghc-typelits-knownnat
+                  , maestro-sdk
+                  , plutus-ledger-api
```

Comment:
Some of the imports seem unnecessary. Can check for warnings in `cabal build all --enable-tests` in new compilation.

Reply:
Done.  Eliminated:
    - MonadRandom
    - plutus-core
    - plutus-ledger-api
    - ply-core
    - random
    - random-fu
    - vector